src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/vedtaksperiode/VedtaksperiodeService.kt

```diff
@@ -505,36 +513,36 @@ class VedtaksperiodeService(
             }
         )
     }
-}
 
-private fun leggTilAvslagsbegrunnelseForUregistrertBarn(
-    avslagsperioder: List<VedtaksperiodeMedBegrunnelser>,
-    vedtak: Vedtak,
-    uregistrerteBarn: List<BarnMedOpplysningerDto>
-): List<VedtaksperiodeMedBegrunnelser> {
-    val avslagsperioderMedTomPeriode = if (avslagsperioder.none { it.fom == null && it.tom == null }) {
-        avslagsperioder + VedtaksperiodeMedBegrunnelser(
-            vedtak = vedtak,
-            fom = null,
-            tom = null,
-            type = Vedtaksperiodetype.AVSLAG
-        )
-    } else {
-        avslagsperioder
-    }
+    private fun leggTilAvslagsbegrunnelseForUregistrertBarn(
```

Comment:
😬 Hvordan ha dette skjedd? Har vi ikke ktlint som en sjekk før vi merger?

Reply:
Ktlint sjekker for indentering, og i dette tilfelle var det riktig siden den var utenfor klassen 😨